argilla_plugins/datasets/hf_sync.py

```diff
@@ -0,0 +1,185 @@
+import dataclasses
+import os
+import time
+from threading import Thread
+from typing import Optional, Set
+
+from datasets import Dataset, load_dataset
+
+import argilla as rg
+from argilla.client.sdk.commons.errors import NotFoundApiError
+from argilla.server.commons.models import TaskType
+
+
+@dataclasses.dataclass
+class Config:
+
+    hf_source: Optional[str] = None
+    hf_target: Optional[str] = None
+    hf_token: Optional[str] = None
```

Comment:
maybe we should be able to set this programmatically via env vars instead of only CLI?

Reply:
Yes. Maybe the Config class can be a pedantic `BaseSettings` class that gives automatic support with env variables.